packages/wallets/src/features/cashier/flows/WalletTransfer/WalletTransfer.tsx

```diff
@@ -1,6 +1,25 @@
-import React from 'react';
+import React, { useCallback, useEffect } from 'react';
+import { useTransferBetweenAccounts } from '@deriv/api';
+import { Loader } from '../../../../components';
 import { TransferModule } from '../../modules';
+import { TransferNotAvailable } from '../../screens/TransferNotAvailable';
 
-const WalletTransfer = () => <TransferModule />;
+const WalletTransfer = () => {
+    const { data, isLoading: isTransferAccountsLoading, mutate } = useTransferBetweenAccounts();
+
+    const requestTransferAccounts = useCallback(() => mutate({ accounts: 'all' }), [mutate]);
+
+    useEffect(() => {
+        requestTransferAccounts();
+    }, [requestTransferAccounts]);
+
+    if (isTransferAccountsLoading || !data?.accounts) return <Loader />;
```

Comment:
for future PRs, question regarding !data?.accounts condition:
how the errors are handled? In case of no accounts returned (so effectively error state) will it be stuck in displaying loader? or will it show some sort of error?

Reply:
hey hey @wojciech-deriv 
all negative scenarios will be handled later in Transfer root component
for now our goal is `happy` transfer